scripts/us_census/pep/monthly_population_estimate/download.py

```diff
@@ -82,13 +82,23 @@ def _save_data(url: str, download_local_path: str) -> None:
                            engine='python',
                            skiprows=17,
                            names=cols)
-        df = _clean_txt_file(df, _SCALING_FACTOR_TXT_FILE)
-        df.to_excel(download_local_path + os.sep + file_name,
+        # Skipping 17 rows as the initial 17 rows contains the information about
+        # the file being used, heading files spread accross multiple lines and
+        # other irrelevant information like source/contact details.
+        df = _clean_txt_file(df)
+        # Multiplying the data with scaling factor 1000.
+        for col in df.columns:
+            if "year" not in col.lower():
+                if _SCALING_FACTOR_TXT_FILE != 1:
```

Comment:
why do we need this? _SCALING_FACTOR_TXT_FILE is a constant

Reply:
Yes agreed this is not needed, we added it just case if we need to add more txt files with other scaling factors in the future. However, this has been removed now.